SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialHandoverManager.cpp

```diff
@@ -0,0 +1,167 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "EngineClasses/SpatialHandoverManager.h"
+
+#include "EngineClasses/SpatialPackageMapClient.h"
+#include "SpatialCommonTypes.h"
+#include "SpatialConstants.h"
+#include "SpatialView/EntityDelta.h"
+#include "SpatialView/SpatialOSWorker.h"
+#include "SpatialView/SubView.h"
+#include "SpatialView/ViewDelta.h"
+
+namespace SpatialGDK
+{
+FSpatialHandoverManager::FSpatialHandoverManager(const FSubView& InActorView, const FSubView& InPartitionView)
+	: ActorView(InActorView)
+	, PartitionView(InPartitionView)
+{
+}
+
+void FSpatialHandoverManager::Advance()
+{
+	for (const EntityDelta& Delta : PartitionView.GetViewDelta().EntityDeltas)
+	{
+		switch (Delta.Type)
+		{
+		case EntityDelta::ADD:
+			PartitionsToACK.Add(Delta.EntityId);
```

Comment:
Seems a safety check that we have authority over this entity would be worthwhile

Reply:
Wouldn't this be part of completeness?